src/pages/Admin/BackgroundSelect/BackgroundSelect.tsx

```diff
@@ -45,45 +45,66 @@ export const BackgroundSelect: React.FC<BackgroundSelectProps> = ({
 
   const hasBackgrounds = !!mapBackgrounds.length && !isLoadingBackgrounds;
 
+  const renderBackground = useCallback(
+    (mapBackground: string, index: number) => (
+      <button
+        className="BackgroundSelect__map"
+        disabled={isUploading}
+        style={{ backgroundImage: `url(${mapBackground})` }}
+        key={index}
+        onClick={() => uploadMapBackground(mapBackground)}
+      />
+    ),
+    [isUploading, uploadMapBackground]
+  );
+
   const renderMapBackgrounds = useMemo(
     () =>
-      mapBackgrounds.map((mapBackground, index) => (
-        <button
-          className="BackgroundSelect__map"
-          disabled={isUploading}
-          style={{ backgroundImage: `url(${mapBackground})` }}
-          key={index}
-          onClick={() => uploadMapBackground(mapBackground)}
-        />
-      )),
-    [isUploading, mapBackgrounds, uploadMapBackground]
+      mapBackgrounds.map((mapBackground, index) =>
+        renderBackground(mapBackground, index)
+      ),
+    [mapBackgrounds, renderBackground]
+  );
+
+  const renderDefaultBackgrounds = useMemo(
+    () =>
+      DEFAULT_BACKGROUNDS.map((mapBackground, index) =>
+        renderBackground(mapBackground, index)
+      ),
```

Comment:
```suggestion
      DEFAULT_BACKGROUNDS.map(renderBackground),
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2327/commits/5f67a155ffeac6b4d7056d782aad631fe5370537